helpers/GSGetter.ts

```diff
@@ -201,6 +203,68 @@ export class GCGetter {
                 const id = await persistence.connect_user_to_calendar_id(calendar[1], context.getSender());
                 const final_calendar_id = await persistence.get_preferred_calendar_id(context.getSender());
                 break;
+
+            case (Command.public):
+
+                const user_id = await read.getRoomReader().getMembers(context.getRoom().id);
```

Comment:
`user_id` is not an accurate name for this variable, as it will contain actually an array of users